cardano-cli/src/Cardano/CLI/Pretty.hs

```diff
@@ -0,0 +1,75 @@
+module Cardano.CLI.Pretty
+  ( Ann,
+    putLn,
+    hPutLn,
+    renderDefault,
+    renderStringDefault,
+
+    black,
+    red,
+    green,
+    yellow,
+    blue,
+    magenta,
+    cyan,
+    white,
+  ) where
+
+import           Control.Exception (bracket_)
+import           Control.Monad.IO.Class (MonadIO, liftIO)
+import           Data.Function (($), (.))
```

Comment:
Why import `Data.Function` and `Prelude`?

Reply:
Fixed.  Thanks!